### PR TITLE
Fix sidebar label dropdown divider

### PR DESCRIPTION
**Pull request by HesterG**
Follow #25312 

Also fix the label dropdown on sidebar

Before:

Screen Shot 2023-06-19 at 08 55 05

After:

Screen Shot 2023-06-19 at 08 57 17

### CODE REVIEW

**Review by lunny — APPROVED** [review]

File: templates/repo/issue/labels/labels_selector_field.tmpl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="ui {{if or (not .HasIssuesOrPullsWritePermission) .Repository.IsArchived}}disabled{{end}} floating jump select-label dropdown">
+<div class="ui {{if or (not .HasIssuesOrPullsWritePermission) .Repository.IsArchived}}disabled{{end}} floating jump select-label label-filter dropdown">
```
Comment (wxiaoguang) [inline templates/repo/issue/labels/labels_selector_field.tmpl]: Don't do this. The `label-filter` styles will pollute this `select-label` dropdown.

Comment (HesterG) [reply templates/repo/issue/labels/labels_selector_field.tmpl]: Thanks, changed to change selector instead in https://github.com/go-gitea/gitea/pull/25359/commits/8ec7f485ca97b474c5406c0c26e57c4a0224bc55

**Review by silverwind — APPROVED** [review]

**Comment by silverwind** [comment]
Can backport if https://github.com/go-gitea/gitea/pull/25312 also backports.

**Comment by silverwind** [comment]
@GiteaBot should try again now that https://github.com/go-gitea/gitea/pull/25372 is merged.

**Pull request merged**